proposals/0220-count-where.md

```diff
@@ -3,8 +3,8 @@
 * Proposal: [SE-0220](0220-count-where.md)
 * Author: [Soroush Khanlou](https://github.com/khanlou)
 * Review Manager: [Ben Cohen](https://github.com/airspeedswift)
-* Status: **Accepted**
-* Implementation: [apple/swift#16099](https://github.com/apple/swift/pull/16099), [apple/swift#22289](https://github.com/apple/swift/pull/22289) (revert PR)
+* Status: **Implemented (Swift 6.0)**
+* Implementation: [apple/swift#16099](https://github.com/apple/swift/pull/16099), [apple/swift#22289](https://github.com/apple/swift/pull/22289), [apple/swift#72705](https://github.com/apple/swift/pull/72705)
```

Comment:
I think we should _either_ identify 22289 as a revert PR here, or only list 72705.